lib/pipedrive/operations/upload.rb

```diff
@@ -0,0 +1,37 @@
+require 'open-uri'
```

Comment:
Is this required? There is possibility to use a url as file. Not sure if this secure enough.

Reply:
maybe it would be better if `#upload` expects a file/IO object, instead of URL/filename?

Reply:
good idea!